.github/workflows/ci-master.yml

```
@@ -0,0 +1,67 @@
+name: CI-master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2-beta
+      with:
+        fetch-depth: 1
```

I believe this is all good with the defaults in v2

```suggestion
    - uses: actions/checkout@v2
```